business/apps.go

```diff
@@ -161,6 +171,14 @@ func (in *AppService) GetAppList(ctx context.Context, namespace string, linkIsti
 				break
 			}
 		}
+
+		if c.Health {
+			appItem.Health, err = in.businessLayer.Health.GetAppHealth(ctx, c.Namespace, appItem.Name, c.RateInterval, c.QueryTime)
```

Comment:
I wonder if it is better to call GetNamespaceAppHealth to collect health from all items and then assign individually in a loop, rather than repeatedly calling GetAppHealth. You would go from N queries to prometheus to a single one for the namespace. Although I don't know if that leads to the same result.

This can be an improvement for another PR if you like the idea. If not... well, nothing to do :-)

Reply:
The point is that now we have a loop to fill all the information for each appItem. We can discuss that and do it in another PR. I think that is another kind of improvement